RNReaderSDK.podspec

```diff
@@ -1,7 +1,7 @@
 
 Pod::Spec.new do |s|
   s.name         = "RNReaderSDK"
-  s.version      = "1.7.2"
+  s.version      = "1.7.4"
```

Comment:
This is the version of the plugin, right?  And it's tracking the iOS SDK version?

That makes sense, I think, but as something to slow-think about later: if we're worried about the lockstep concerns, we might make the plugins have a very different numbering scheme (I'm thinking date-based, i.e. RNReaderSDK:2024-09-18).

Reply:
this should also be 1.7.7 because IIUC this is the version of the plugin itself